docs/developer/examples/custom_components.js

```diff
@@ -0,0 +1,220 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
+/* eslint-disable strict */
+/**
+ This is a demo custom component used to try out the feature OR to run tests.
+ It is a simple button that will modify the value value to 'test'. Un-minified jsx code below.
+  export default function RegionSelector(props) {
+  const { param, handleBasicFormParamChange } = props;
+
+  const [value, setValue] = useState(props.value || "");
+
+  useEffect(() => {
+    setValue(props.param.value);
+  }, [props.param.value]);
+
+  const handleChange = newValue => {
+    handleBasicFormParamChange(props.param)({
+      currentTarget: {
+        value: "test"
+      }
+    });
+    setValue(newValue);
+  };
+
+  const selectedStyle = {
+    backgroundColor: "rgba(0, 140, 255, 0.19)"
+  };
+
+  return (
+    <>
+      <button
+        key={`test`}
+        type="button"
+        onClick={() => handleChange("test")}
+        style={"test" === value ? selectedStyle : null}
+      >
+        Test
+      </button>
+    </>
+  );
+}
+ */
+(function(e, a) { for(var i in a) e[i] = a[i]; }(exports, /******/ (function(modules) { // webpackBootstrap
```

Comment:
can we just use non-minified code here? (or at least tell instructions to generate the minified version from the original code). 

The idea is to allow people to use this code as template for future components, if it's minified it won't be that helpful.

Reply:
I added 3 different examples. A `jsx` component, that `jsx` component converted to `js` and then finally the minified version of that file as generated by the remote helper library